asm/internal/ll.bnf

```diff
@@ -1610,7 +1617,7 @@ UnnamedAddr
 
 OptCommaAlign
 	: empty
-	| "," Align
+	| "," Align CommaAttachedMDs
```

Comment:
Sorry, this was a bit of a kludge, but again trying to work around LR1 limitations. I'll add the comments for now at least.